config/locales/en.yml

```diff
@@ -141,6 +141,19 @@ en:
     values:
       true: "Yes"
       false: "No"
+    states:
+      label:
+        draft: Draft
+        awaiting_2i: Awaiting 2i
+        scheduled: Scheduled
+        published: Published
+        superseded: Superseded
+      colour:
+        draft: yellow
+        awaiting_2i: turquoise
+        scheduled: light-blue
+        published: green
+        superseded: orange
```

Comment:
This is a loosely held opinion, but I'm not sure colours belong in a translation file - a better place might be a helper mehtod.

Reply:
I hadn't thought about this but yeah that makes sense. If we needed to make a Welsh `cy.yml` file the values would be the same.

Reply:
thanks both, I've rejigged this so that the component now handles the colours and the locale file translates the `Edition#state` to a user-facing "status" shown in the tag.